packages/dash.rb

```diff
@@ -2,12 +2,12 @@
 
 class Dash < Package
   description 'The Debian Almquist Shell (dash) is a POSIX-compliant shell derived from ash that executes scripts faster than bash and has fewer library dependencies.'
-  homepage 'https://salsa.debian.org/debian/dash'
+  homepage 'https://invisible-island.net/mawk/'
```

Comment:
Why the homepage change?  This is mawk, not dash.

Reply:
because i'm dumb and my clipboard is worse 🤕